aea/protocols/generator/generator.py

```diff
@@ -26,12 +26,11 @@
 from datetime import date
```

Comment:
Better to call it `base.py` to align with our naming conventions.

Reply:
Done